Automattic/wp-calypso
# WooCommerce: Remove unused state, components and libs

#### Changes proposed in this Pull Request

Since the WooCommerce store was moved to wp-admin and we added a deprecation notice in Calypso, we've been removing the old unused WooCommerce store app code, piece by piece.

This PR removes a big chunk of the unused state, libraries, and components, as well as unused WooCommerce Services code, leaving essentially what looks like to currently be used by the deprecation notice and the store stats that are still working and accessible. 

#### Testing instructions

* Verify nothing we're removing is used. That can take a while manually, but the following instructions can help too 😅 
* Verify Calypso still builds well.
* Verify smoke testing Calypso reveals no problems.
* Verify `/store/:site` for a site with WooCommerce still displays the deprecation notice properly.
* Verify `/store/stats/orders/day/:site` for a site with WooCommerce still works well (smoke test some store stats sections).
* Verify tests still pass.
